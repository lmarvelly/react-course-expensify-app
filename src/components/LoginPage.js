import React from 'react';

const LoginPage = () =>
(
	<div>
		<form>
			<input type='text' placeholder='Username' />
			<input type='password' placeholder='Password' />
		</form>
	</div>
);

export default LoginPage;