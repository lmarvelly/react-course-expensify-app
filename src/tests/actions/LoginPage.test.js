import React from 'react';
import { shallow } from 'enzyme';
import LoginPage from '../../components/LoginPage';

// Stateless functional component
test('should render LoginPage correctly', () =>
{
	const wrapper = shallow(<LoginPage />);
	expect(wrapper).toMatchSnapshot();
});