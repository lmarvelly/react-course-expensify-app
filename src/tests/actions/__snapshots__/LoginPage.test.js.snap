// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LoginPage correctly 1`] = `
<div>
  <form>
    <input
      placeholder="Username"
      type="text"
    />
    <input
      placeholder="Password"
      type="password"
    />
  </form>
</div>
`;
